tests/css-epub-word-break/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,20 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>EPUB CSS Word Break</title>
+  </head>
+  <body>
+    <h1>-epub-word-break Tests</h1>
+
+    <p>1. This test passes if all of the words in the following block of text appear without breaks.</p>
```

Comment:
I think you should clarify what a break is. "If none of the words in the following block of text wrap to the next line mid-word"?